### ISSUE TITLE
Multi-line (or other formatting) within an alert 

### ISSUE THREAD

**Issue by ajaymills** [sandialabs/scot#25]
Hi, I've successfully been able to add a custom alert parser, and I can see the multiple named columns that it creates in the GUI when such an alertgroup is selected. 

With in a column of the alert, is there a way to make multi-line text look nice? When displayed, it appears to drop newlines, and also  \<br/\> is printed out literally.

**Comment by toddbruner** [comment]
We are currently working on the next version that will revamp this and unfortunately, will probably not be able to devote a lot of time looking into this.  The code that currently renders the alert table is in /public/event.js in the function "render_alert_body".  The few moments I looked at it, I don't see anything off-hand that we are doing to prevent html interpretation.  When your alert parser is creating alerts, there may be something that may be "encoding" the html entities.  That would explain the literal '<br/>'.   If you find the problem let me know, or if you want to send a pull request, that would be great as well.

**Comment by UmiDevSec** [comment]
We don't assume that alert parsing properly sanitized the alert. Improper sanitizing could lead to XSS. 

If you look at [line 873](https://github.com/sandialabs/scot/blob/master/public/event.js#L873) of commit fc5e08207d58d09043deaa7a3e1fe1d03ba95307 you'll see that we're setting the <th> element's text field to prevent any interpretation of HTML. 

If you just want to turn \<br> into newlines on the screen, then add the following code right after line 873.  This will make all the \<br> display properly.
```
key.html(key.html().replace('&lt;br&gt;', '<br>'));
```

Note, I only tested that code in jsFiddle, no guarantee it'll work

**Comment by ajaymills** [comment]
Hi, for some reason it didn't work on my server. After looking at the JS, although I'm not that familiar with the language it looks like that section is building the header for the table, not the internal cells. 

I noticed down on line 928 we have this:

>  value = value.replace(/\n/g, '\<br /\>');

It looks like there was an attempt to do breaks  before, but for some reason this line has no effect  either. Maybe the newlines are already stripped out by that point in the code. 

I replaced that line by this:

> value=value.replace(/&lt;br&gt;/g, '<br/>');

And now my \<br\> are properly rendered in the HTML output.
